src/cli/peakdetector/peakdetector.pro

```diff
@@ -14,7 +14,7 @@ QMAKE_CXXFLAGS += -std=c++11
 
 INCLUDEPATH +=  $$top_srcdir/src/core/libmaven  $$top_srcdir/3rdparty/pugixml/src $$top_srcdir/3rdparty/libneural $$top_srcdir/3rdparty/libpls \
 				$$top_srcdir/3rdparty/libcsvparser $$top_srcdir/3rdparty/libdate $$top_srcdir/3rdparty/libcdfread \
-				$$top_srcdir/src/pollyCLI $$top_srcdir/3rdparty/obiwarp
+				$$top_srcdir/src/pollyCLI $$top_srcdir/3rdparty/obiwarp $$top_srcdir/src/core/libmaven/alignment
```

Comment:
@TanmayGoyal No need to include the path of alignment folder.

Reply:
Done, removed alignment folder path from all project files